docs/workflows-config.md

```diff
@@ -142,16 +142,16 @@ executors.
 actions:
   - name: "Test all targets (Mac)"
     os: "darwin" # <-- add this line
-    arch: "arm64" # <-- add this line for M1 Macs only
+    arch: "arm64" # <-- add this line for Apple silicon (M-series) only
```

Comment:
```suggestion
    arch: "arm64" # <-- add this line for Apple Silicon (M-series) only
```

Reply:
Apple's docs have it lowercased: https://support.apple.com/en-us/HT211814
Wikipedia too: https://en.wikipedia.org/wiki/Apple_silicon

Reply:
Interesting. TIL.